plugins/content/caption/base/lib/captionManagers/srtCaptionsContentManager.php

```diff
@@ -139,4 +139,40 @@ public function merge($content, $toAppend)
 	{
 		return $content . $toAppend;
 	}
-}
\ No newline at end of file
+
+	public function createCaptionsFile($content, $clipStartTime, $clipEndTime, $timeCode, $globalOffset)
+	{
+		static $newLineIndex = 1;
```

Comment:
Why do we need it as static, since its only being referenced here and only within this function it can be local var or member of the class if you need to keep it between the iterations

Reply:
Tried to set it as a property of the class - but it's being reset every time we build a new 'chunk' of SRT
When load the class at: plugins/content/caption/base/batch/CopyCaptions/KAsyncCopyCaptions.class.php:159